README.md

```diff
@@ -83,6 +84,7 @@ postgresql_users:
   - name: baz
     pass: pass
     encrypted: no       # denotes if the password is already encrypted.
+    state: present      # optional; one of 'present', 'absent'
```

Comment:
If this is a string, then we should double-quote it.  In general booleans are unquoted, but all strings should be quoted.

Reply:
Sounds good. Does it make sense to make the other string values double quoted in the README too?